R/galah_download.R

```diff
@@ -6,85 +9,72 @@ library(here)
 library(APCalign)
 library(skimr)
 
-# Configure galah
-galah_config(email = Sys.getenv("ALA_EMAIL"),
-             atlas = "Australia")
+download_ala_obs <- function(taxa = "Plantae", output_dir = "infinity-app/data/") {
+  
+  # 1. Data retrieval
+  NSW_obs <- retrieve_data(taxa)
```

Comment:
Nit picking here, maybe remove NSW form assigned object if we are downloading for whole of Australia